python/pyspark/ml/classification.py

```diff
@@ -454,7 +454,8 @@ class TreeClassifierParams(object):
     impurity = Param(Params._dummy(), "impurity",
                      "Criterion used for information gain calculation (case-insensitive). " +
                      "Supported options: " +
-                     ", ".join(supportedImpurities), typeConverter=TypeConverters.toString)
+                     ", ".join(supportedImpurities) + ". (Default: gni)",
```

Comment:
It hasn't been on the python side, but the Scaladoc includes mention of the default values so I've been adding it as I go through them anyways (see https://github.com/apache/spark/pull/12914 for the shared params case)